recipes/cohesion/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cohesion" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mschwager/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: abb5a88b70de96e7dfb65244cc13cff7e9c5edee5333175d5a7668ac7c742c6c
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - cohesion = cohesion.__main__:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  commands:
+    - pip check
+    - cohesion -h
+    - "flake8 --version | grep cohesion"
+  requires:
+    - flake8
+    - m2-grep  # [win]
```

Comment:
`noarch: python` packages are only built on Linux. But I guess this doesn't really hurt either.

Reply:
Doesn't hurt to capture that when it's as easy as this.